src/vw/src/test/scala/VerifyVowpalWabbitRegressorFuzzing.scala

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark
+
+import org.apache.spark.ml.util.MLReadable
+import org.apache.spark.sql.{Column, DataFrame}
+
+class VerifyVowpalWabbitRegressorFuzzing extends EstimatorFuzzing[VowpalWabbitRegressor] {
```

Comment:
This should be in your regular test class as opposed to another test class

Reply:
I separated it as it made testing it individually really hard. Maybe I'm just missing something in IntelliJ?

Reply:
Okay! This is good feedback